profile.html

```diff
@@ -72,7 +72,7 @@ <h2 id="title">Settings</h2>
 
     <nav class="navigation">
       <ul class="navigation__list">
-        <li class="navigation__list-item">
+        <li class="navigation__list-item >
```

Comment:
the closing quotes are missing:

```suggestion
        <li class="navigation__list-item">
```